scripts/travis/script_build.sh

```diff
@@ -19,6 +19,8 @@ if [[ ${PLATFORM} == "ios" ]]; then
     # Build ios project
     echo "Building ios project (simulator)"
     make ios-sim
```

Comment:
Looks like you probably should disable the ios-sim build task here since it's causing travis builds to go too long and terminate the job.